### ISSUE TITLE
[BUG] Not Syncing for days now. 

### ISSUE THREAD

**Issue by IronM6n** [Chia-Network/chia-blockchain#5939]
**Describe the bug**
The Node is not syncing. The height is not matching status. its been doing this for days. 

**Desktop (please complete the following information):**

- OS: [e.g. Linux]Windows 
- OS Version/Flavor: [e.g. CentOS 7.2] 10 Pro 20H2
- CPU: [e.g. Intel Xeon 8175M]Intel(R) Core(TM) i5-4250U CPU @ 1.30GHz   1.90 GHz

**Additional context**

![Capture](https://user-images.githubusercontent.com/84716917/119345483-f18a7080-bc66-11eb-913d-a16af420770c.PNG)

**Comment by zuzolo** [comment]
Same situation since 1.1.6 update, syncing status but no progress, no problem with 1.1.5

**Comment by EvilCrust** [comment]
Same issue, not able to sync for days. Managed to sync once for many 20 minutes.  How is anyone excepted to farm or do anything other than create plots ?  I've tried system reboots, force reload, run CLI mode, and GUI mode with no success. Yes, my system time is correct, and I've connected to introduce-va.chia.net and node.chia.net.  Nothing seems to work. It's very frustrating to not be able to sync of maintain sync. I a week 20 minutes of synced time is just not acceptable.

**Comment by siaarzh** [comment]
Duplicate of #3298

See https://github.com/Chia-Network/chia-blockchain/issues/3298#issuecomment-845414120, https://github.com/Chia-Network/chia-blockchain/issues/3298#issuecomment-830738275 for possible solutions

**Comment by IronM6n** [comment]
thanks Siaarzh, I will try this once my plotting is done and respond back if this resolved the issue.

**Comment by IronM6n** [comment]
after several reboots and deleting peer_table_node.sqlite.  fixed the issue
